feat(serverless): changelog for startup probe (#6812)
scaleway/docs-content: changelog/august2026/2026-08-21-serverless-containers-startup-probe-in-console.mdx

---
title: Serverless Containers - Startup probe configuration available in the console
status: added
date: 2026-08-21
category: serverless
product: containers
---

> Startup probe configuration for [health checks](/serverless-containers/concepts/#health-check) is now available in the console.

Reviewer: ```suggestion
You can now configure startup probes for [health checks](/serverless-containers/concepts/#health-check) directly in the console.
```
